app/(gcforms)/[locale]/(app administration)/admin/(with nav)/settings/[settingId]/page.tsx

```diff
@@ -0,0 +1,35 @@
+import { serverTranslation } from "@i18n";
+import { requireAuthentication } from "@lib/auth";
+import { checkPrivilegesAsBoolean } from "@lib/privileges";
+import { Metadata } from "next";
+import { ManageSettingForm } from "../components/server/ManageSettingForm";
+import { Suspense } from "react";
+import Loader from "@clientComponents/globals/Loader";
+
+export async function generateMetadata({
+  params: { locale },
+}: {
+  params: { locale: string };
+}): Promise<Metadata> {
+  const { t } = await serverTranslation("admin-settings", { lang: locale });
+  return {
+    title: `${t("title-update")}`,
+  };
+}
+
+export default async function Page({ params: { settingId } }: { params: { settingId: string } }) {
+  const { user } = await requireAuthentication();
+  checkPrivilegesAsBoolean(user.ability, [{ action: "update", subject: "Setting" }], {
+    redirect: true,
+  });
+  const { t } = await serverTranslation("admin-settings");
+
+  return (
+    <>
+      <h1 className="border-0 mb-10">{t("title-update")}</h1>
+      <Suspense fallback={<Loader />}>
+        <ManageSettingForm settingId={settingId} canManageSettings={true} />
```

Comment:
I wanted to remove it but thought it could be useful in the future if a user can only view but not change settings. Though would this ever be the case? Thinking about it now, probably not. 
Unless that's not the case, I'll remove the canManageSettings logic?

Reply:
How about replacing the constant with a result from `checkPrivilegesAsBoolean` instead?  This way it's a very small lift to gain that functionality of allowing a user to view.

Reply:
On second thought, having that logic could be confusing. I removed it